modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -4080,9 +4069,6 @@
                                   ]
                                 }
                               },
-                              "hasSeparationPay": {
```

Comment:
I'm guessing we're going to want to have some release notes that mention this and other deprecated fields?